devito/types/sparse.py

```diff
@@ -650,7 +650,7 @@ def interpolate(self, expr, offset=0, increment=False, self_subs={}):
         idx_subs, temps = self._interpolation_indices(variables, offset)
 
         # Substitute coordinate base symbols into the interpolation coefficients
-        args = [expr.subs(v_sub) * b.subs(v_sub)
+        args = [expr.xreplace(v_sub) * b.xreplace(v_sub)
```

Comment:
remind me why? I guess you've told me already...but I forgot

Reply:
subs has sooooo many cases and I don't like it.
xreplace is nice and easy and is implemented in `Differentiable`